internal/entities/properties/properties.go

```diff
@@ -18,12 +18,12 @@ package properties
 
 import (
 	"fmt"
+	"iter"
```

Comment:
FWIW this was the old linter not recognizing `iter` as being from the go stdlib..

Reply:
I added the nolint so we unblock main